2-mandatory/2-water-bottle.js

```diff
@@ -19,18 +19,25 @@ let bottle = {
   volume: 0,
   fillUp: function () {
     // calling this function should completely fill your bottle (volume = 100);
+    const spaceToFill = 100 - this.volume;
+    this.volume < 100? this.volume += spaceToFill : this.isFull()
```

Comment:
Usually, we use a ternary operator as an expression (where we care about the result). The code here is still functionally working - but there's a couple of problems with this in my opinion:
1. `this.isFull()` is being called even though it's not needed here
2. While the code is more concise, I think it's a little bit less readable. You are using the ternary operator - but throwing the result away (nothing is being returned from the function). This can be a bit confusing to other developers reading the code.
Feel free to let me know if you'd like me to explain more, or if this comment is confusing 😄